README.md

```diff
@@ -156,18 +156,10 @@ Some hotkeys might already be used by GNOME Shell - please check your keybinding
 | Arch Linux   | AUR         | From latest release: `yay -S gnome-shell-extension-material-shell` |
 | Fedora       | DNF         | `sudo dnf install gnome-shell-extension-material-shell` |
 | NixOS        | nix         | `nix-env -i gnome-shell-extension-material-shell` |
-| Others       | source      | `git clone https://github.com/material-shell/material-shell.git ~/.local/share/gnome-shell/extensions/material-shell@papyelgringo` |
+| Others       | source      | `git clone https://github.com/material-shell/material-shell.git && cd material-shell && make install` |
 
 (We appreciate package maintainers! If you would like to make a package available for your distro please submit a PR so it can be added here!)
 
-3. Reload GNOME Shell:
-  + On X.org: Hit `Alt+F2` and type the command `r`
-  + On Wayland: Log out and back in
-  
-4. Open `gnome-tweaks` and activate the `Material Shell` extension **OR** enable it using 
-```bash
-gnome-extensions enable material-shell@papyelgringo
-```
```

Comment:
this shouldn't be deleted

Reply:
The `make install` script will run this automatically. For wayland it will instruct the user to log out and back in again.